server/form-pages/utils/questions.ts

```diff
@@ -523,6 +523,10 @@ export const getQuestions = (name: string) => {
           answers: yesOrNo,
         },
         servicesDetail: { question: 'Please state which services.' },
+        isEngagedWithServicesInCustody: {
```

Comment:
Should we also add to the test fixtures `applicationData.json` and `applicationDocument.json`?

Reply:
oh yes, missed this! thanks